controls/datetimepicker/styling/popup-styling.md

```diff
@@ -14,8 +14,8 @@ By using the `PopupSettings` property (of type `Telerik.Maui.Controls.PickerPopu
 * `HeaderStyle`(of type `Style` with target type `telerik:PickerPopupHeaderView`)&mdash;Defines the popup header style.
 * `HeaderLabelStyle`(of type `Style` with target type `Label`)&mdash;Defines the popup header label style.
 * `FooterStyle`(of type `Style` with target type `telerik:PickerPopupFooterView`)&mdash;Defines the popup footer style.
-* `AcceptButtonStyle`(of type `Style` with target type `Button`)&mdash;Defines the **Accept** button style.
-* `CancelButtonStyle`(of type `Style` with target type `Button`)&mdash;Defines the **Cancel** button style.
+* `AcceptButtonStyle`(of type `Style` with target type `Button`)&mdash;Defines the `Accept` button style.
```

Comment:
This comment is for all list items: place a space between the property and the opening parenthesis. For example:
`AcceptButtonStyle` (of type..